database/migrations/2021_03_03_100000_create_zip_codes_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateZipCodesTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('zip_codes', function (Blueprint $table) {
+            $table->string('code', 5)->unique()->primary(); // Actual ZIP Code. Has to be string because can have leading zeros. Check model to see how it is made Primary Key.
+            $table->string('timezone')->nullable();
+            $table->decimal('latitude', 4, 2)->nullable();
+            $table->decimal('longitude', 5, 2)->nullable();
+            $table->string('county')->nullable();
+            $table->foreignIdFor(app('state'));
+            $table->foreignIdFor(app('city'), 'city_id');
+            $table->foreignIdFor(app('city'), 'city_alternate_1')->nullable();
+            $table->foreignIdFor(app('city'), 'city_alternate_2')->nullable();
+            $table->foreignIdFor(app('city'), 'city_alternate_3')->nullable();
+            $table->foreignIdFor(app('city'), 'city_alternate_4')->nullable();
```

Comment:
Let's use a many-to-many relationship between City (#18) and ZIP (#17) instead of this old database approach I created for it.